roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -165,6 +165,13 @@
   retries: 20
   delay: 15
 
+- name: Adding Kibana route information to loggingPublicURL
+  modify_yaml:
+    dest: "{{ openshift.common.config_base }}/master/master-config.yaml"
+    yaml_key: assetConfig.loggingPublicURL
+    yaml_value: "https://{{ openshift_logging_kibana_hostname }}"
+  notify: restart master
```

Comment:
could we leverage the include_role for this in the playbook and just call the specific task to update the master-configs to do this?